service/deployer/eventer/github/client.go

```diff
@@ -0,0 +1,163 @@
+package github
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	microerror "github.com/giantswarm/microkit/error"
+)
+
+const (
+	// deploymentUrlFormat is the string format for the GitHub
+	// API call to fetch Deployments.
+	deploymentUrlFormat = "https://api.github.com/repos/%s/%s/deployments"
+
+	// deploymentStatusUrlFormat is the string format for the
+	// GitHub API call to post Deployment Statuses.
+	deploymentStatusUrlFormat = "https://api.github.com/repos/%s/%s/deployments/%v/statuses"
+
+	// etagHeader is the header used for etag.
+	// See: https://en.wikipedia.org/wiki/HTTP_ETag.
+	etagHeader = "Etag"
```

Comment:
I kept them private because they're only used internally to the package, I didn't really see why they'd need to be outside, even for godoc.